<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="pbxcore"
           tests="17"
           failures="3"
           errors="0"
           skipped="0"
           time="392.099">
  <properties />
            <testcase name="B_addPin" time="18.437" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.PINList">
                <failure type="java.lang.AssertionError"
                  message="【预期：ABCDE~*_-+.?123456789012345678901234567890~~~》《》？、“：+-——）（*……）” 实际：test2】"
              >
<![CDATA[java.lang.AssertionError: 【预期：ABCDE~*_-+.?123456789012345678901234567890~~~》《》？、“：+-——）（*……）” 实际：test2】
  com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
  com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:579)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:142)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:116)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:199)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:210)
  com.yeastar.swebtest.driver.YSMethod.YS_CallFeature.addPinList(YS_CallFeature.java:307)
  com.yeastar.swebtest.testcase.DetailCase.pbxcase.PINList.B_addPin(PINList.java:53)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.run(TestNG.java:1048)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
            <testcase name="G_checkPin" time="44.533" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.PINList">
                <failure type="java.lang.AssertionError"
                  message="外线呼出进入pinList 【预期：true 实际：false】"
              >
<![CDATA[java.lang.AssertionError: 外线呼出进入pinList 【预期：true 实际：false】
  com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
  com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:579)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:142)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:116)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:333)
  com.yeastar.swebtest.testcase.DetailCase.pbxcase.PINList.G_checkPin(PINList.java:124)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.run(TestNG.java:1048)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
            <testcase name="E_checkPin" time="42.291" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.PINList">
                <failure type="java.lang.AssertionError"
                  message="外线呼出进入pinList 【预期：true 实际：false】"
              >
<![CDATA[java.lang.AssertionError: 外线呼出进入pinList 【预期：true 实际：false】
  com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
  com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:579)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:142)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:116)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:333)
  com.yeastar.swebtest.testcase.DetailCase.pbxcase.PINList.E_checkPin(PINList.java:93)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.run(TestNG.java:1048)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="K1_recovery_InRoute" time="27.692" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.PINList" />
                <testcase name="J1_deletePartPin_no" time="14.391" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.PINList" />
                <testcase name="J3_deleteAllPin_no" time="9.823" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.PINList" />
                <testcase name="A_addExtension" time="12.126" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.PINList" />
                <testcase name="F_deletePinNumber" time="20.006" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.PINList" />
                <testcase name="J2_deletePartPin_yes" time="13.344" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.PINList" />
                <testcase name="H2_deleteOnePin_yes" time="24.784" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.PINList" />
                <testcase name="D_editInRoute" time="34.107" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.PINList" />
                <testcase name="H1_deleteOnePin_no" time="16.820" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.PINList" />
                <testcase name="I_checkPin" time="36.762" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.PINList" />
                <testcase name="J4_deleteAllPin_yes" time="13.307" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.PINList" />
                <testcase name="K2_recovery_Pin" time="10.005" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.PINList" />
                <testcase name="C_editDISA" time="29.151" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.PINList" />
                <testcase name="K3_recovery_Disa" time="24.520" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.PINList" />
      
</testsuite>
